dashboard-frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,42 @@
+import useAuth0 from '../../hooks/useAuth0';
+import PrimaryButton from '../../components/PrimaryButton';
+import { ReactComponent as Logo } from '../../assets/logo.svg';
+import { ReactComponent as WelcomeIllustration } from '../../assets/launching-illustration.svg';
+
+const LoginPage = () => {
+  const { loginWithRedirect } = useAuth0();
+
+  return (
+    <div className="h-screen flex bg-gray-100 justify-center pt-12">
+      <div>
+        <div className="flex justify-center pb-8">
+          <Logo className="h-12 self-center" title="Virtual Lab logo" />
+        </div>
+
+        <div className="bg-white shadow overflow-hidden rounded-none sm:rounded-md pt-8 pb-10">
+          <div className="flex flex-col items-center">
+            <WelcomeIllustration className="h-56 w-72" />
+
+            <p className="pt-10 pl-9 pr-9 sm:pl-20 sm:pr-20 text-4xl text-gray-900 text-center">
+              Ready to start?
+            </p>
+
+            <p className="pt-5 pl-9 pr-9 sm:pl-20 sm:pr-20 text-xl text-gray-900 text-center">
+              Log in now to view the progress of your Studies!
```

Comment:
Will have to work on this copy at some point but it's good for now ;) (they will also be able to launch studies, not just view the progress!!)

Reply:
Uups yes you're right, I'll add this to my todo list.

Thanks 👍